lib/superstore-sync.js

```diff
@@ -14,15 +14,19 @@ var store = {};
 var persist = true;
 
 // Watch for changes from other tabs
-window.addEventListener("storage", function(e) {
-  if (keys[e.key]) {
-    keys[e.key] = true;
-    store[e.key] = JSON.parse(e.newValue);
-  }
-});
+
 
 function Superstore(type) {
 	this.storage = window[type];
+  this.keys = {};
+  this.store = {};
+
+  window.addEventListener("storage", function(e) {
```

Comment:
Nope - although it will have a storageArea property which will be different

https://developer.mozilla.org/en-US/docs/Web/Events/storage